### ISSUE TITLE
cant post photo in inbox

### ISSUE THREAD

**Issue by jhayden2000** [SocialEngine/phpv4-issues#699]
(Thanks for reporting an issue! Please make sure you click the link above to view the issue guidelines, then fill out the blanks below.)

What are the steps to reproduce this issue?
-------------------------------------------
1. …start an inbox conversation
2. ... when the receiver replies, have them attach a pic
3. …

What happens?
-------------
… you are not able to click on the send reply button

What were you expecting to happen?
----------------------------------
… to be able to send the message

Any logs, error output, etc?
----------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.) 

```
Paste Log Here
```

Any other comments?
-------------------
…

What versions of software are you using?
----------------------------------------
**Operating System:** …

**SocialEngine PHP Version:** … **4.9**
![inboxphotos](https://cloud.githubusercontent.com/assets/20259022/25300420/f8dc41b6-26dc-11e7-858f-225fcd61029f.png)

**Comment by Elshara** [comment]
I've been having problems with post attachments as well. They simply
fail to show up.

**Comment by Khushboo016** [comment]
Hi @jhayden2000, 

Regarding "you are not able to click on the send reply button":
You cannot send an empty message having only an attachment, please also write some text in the message body and then try to click "Send Reply" button, you'll see message will be sent to the receiver along with the image attached.

But yes, there is some UI issue the image attachment is not shown properly before sending the message and is shown cropped. We've added the issue to our internal bug tracker and this will be fixed soon.

**Comment by Elshara** [comment]
Cool :)

**Comment by DonnaScriptTechs** [comment]
Please test this in 4.9.1 once we release it as my test just showed super big images in my inbox (as I had attached super big images to test). I think this one might be resolved but would appreciate your feedback once we have 4.9.1 released. Thank you.

**Comment by jhayden2000** [comment]
Donna you work with social engines too?

**Comment by DonnaScriptTechs** [comment]
Hi! Yes I do. :) I've been working here for several months now as Chief Evangelist.

**Comment by jhayden2000** [comment]
Good for you. So no more phpfox work? What made you move?

**Comment by DonnaScriptTechs** [comment]
Ray and I both left. https://forum.scripttechs.com/discussion/137/my-resignation-from-phpfox-and-new-opportunities and https://forum.scripttechs.com/discussion/144/i-ve-resigned-from-phpfox  . We are both happy to be here.  Feel free to pm me at my site or at the SE Community.

**Comment by Elshara** [comment]
Yeah it's really a blessing to have you here. All of you. You make so
much happen with us it truly feels like we come to life by our
interactions for Social Engine. I hope all is settling nicely.
In other words, keep fighting the good fight and I hope you are doing well.
@1ambda
@abhimanyubind
@accolver
@adi90x
@adira-g
@AgtOrange
@ajbasket
@Ak-Army
@akilivan
@albertpak
@alex2211
@alimoosavi15
@alx2bgx
@ammachado
@angelflo
@angelinfante
@Anshuse
@antmanler
@Argentum88
@arundevr
@asgaines
@ashish-gadiya-bs
@AshtonRoute
@aslafy-z
@auntialias
@ayhanbastan
@badc0re
@balramgoyal
@bamboo81
@beatcracker
@beevcra
@Ben2522
@bfriended
@bigpaulie
@BikersPost
@billhibazzz
@birkof
@bkuhl
@blake83
@boristimp
@brettneese
@brian7997
@burdiyan
@Burno317
@carlomagno83
@caseynally
@cgcb
@chrisjacob23
@ciaoamigo
@ckreutz
@CLG-Brainstorm
@clok
@ClubbinHeaven247
@codedevote
@coder4web
@CostaWomen
@crazytechjunkie
@crissi
@cristianogregnanin
@crossy4life
@dabeck
@dalareo
@Danno040
@darkasylumradio
@darrenscerri
@dave1966uk
@davidfavor
@davidjericho
@daxingplay
@ddelnano
@denji
@diceone
@dirkdevriendt
@djcenox
@dolinsky
@Domokun
@DonnaScriptTechs
@dqtranmd
@dreamgeekcoder
@drpebcak
@dsci
@duck1123
@dvital
@eduardotamaki
@ekandreas
@ellerbrock
@Elshara
@ephillipe
@ErikHP
@ernie003
@eshork
@etlweather
@eugene-wh
@Fendt1
@ferjflores
@FG-QuirkyKitty
@fineh
@fire228
@fixe
@florentb
@francoishill
@FrankatMassif
@freakerparty
@fredrikwillers
@frleb
@FurNation
@gabelul
@GabryNet
@garciagabriel
@GarfieldS
@ghost
@gjohnson
@grachevko
@grappendorf
@gsf00001
@guersam
@guigo2k
@gustavo40
@happycoder1337
@happylinks
@hasithaaa
@haswalt
@Hidekikun
@Hostile
@Hostiso
@huseyin-cakmak
@iborisovBG
@IgArI0K
@inkyj
@jacyzon
@jaketbouma
@jamesoclark
@jamesonw
@jasoncormier
@javatech20
@JayBizzle
@JazzJane
@jbakuk
@jbellomy
@jdouk69
@jeffersonsouza
@jhayden2000
@jim55
@jimakker
@jkwas
@jlopez3573
@jmloeffler
@Joeasaurus
@jonathanbloom
@jonathandietz
@JonoB
@JonSimon93
@jurajseffer
@justsml
@k2n
@kain2142
@kanchang
@keshavmd
@kevin-thorpe
@Khushboo016
@kingmxiii
@kristofferahl
@larsha
@Lavie7
@lavvy
@leandrotavaresrocha
@leifg
@LewisW
@lovettmade
@LuciusPrime
@luramos2016
@marcovn
@marcusfsza
@mathuin
@mauchede
@maxx24
@maZahaca
@mbajur
@mbwalsh
@meetgerry
@MeisterG
@MeliMolo
@merlin1996
@MiamiShade
@michaelcurry
@michaellopez
@Mirabis
@mitcholi
@miv
@mjaverto
@mlosev
@MMAMELEE
@mohitg-bs
@mschroeder223
@Mthorganby
@mujaksaba
@muratyamac
@myetter37
@nateritter
@NaughtySocial
@NaughtyToWild
@nayena
@nbicalcarata
@nbrohee
@netanelgilad
@netojose
@niranjanmangal
@nizhnyk
@nk45
@non4eg
@nunofgs
@nviitala
@oddityFry
@orikook
@pacodrums
@PaloPepper
@patrickbard
@patrys666
@PennyRoomHost
@personian
@pib
@piotron
@pjfsilva
@playmusician
@poeticjustice
@princeade
@pschrammel
@pulsatemedia
@qulo
@raphink
@rattfieldnz
@RaymondBenc
@rfink
@rkrathor
@robbydooo
@roflmao
@Rucknar
@rvichery
@rwetzlmayr
@sagekalmus
@saidimu
@scrutinizer-auto-fixer
@sdurrheimer
@SEKyle
@sergeng
@sgreiner
@shannonlmann
@sharemyguitar
@shivam-chaturvedi-bs
@shivani2910
@simon01100001
@simpliby
@skidder
@skylucrew
@SlavaAurim
@smarty005
@smarty555
@smazurov
@socialengineaddons
@SocialEngineMarket
@Spaceinvaderz
@Spielmitmir2003
@SpiritWolfie
@splazit
@sra
@sstine
@staegi
@stewartwebb
@stewfoxdev
@sunfellow
@SuperGlobal
@Suxxen
@swishemart
@tbermudas
@thatspeedykid
@TheGreatAlexander
@thespidirman
@tkshredder
@Tomislav20
@topikiagora
@torrottum
@Traveler0
@travelinks
@TravisHollister
@troydbertrand
@V-gr
@vadimline
@Vaibhavagarwal0807
@verdel
@vinitkh
@viveksharma01
@vjgarg2
@Voxitt
@vpaliwal
@wadkins
@warroyo
@whiteadam
@wizzro
@wojtekk
@wtoalabi
@XuanCHC
@xyzsimon
@y2k4ever
@yunlzheng
@Yunod
@YuriyIlyin
@zachsjc
@zamanmukhtar66
@zambon
@Zenithar
@zzswang

**Comment by DonnaScriptTechs** [comment]
Thanks @Elshara

**Comment by 1ambda** [comment]
Hi,  @Elshara Why i was mentioned here?

**Comment by Elshara** [comment]
You're welcome.

**Comment by Mthorganby** [comment]
@jhayden2000
  
A temporary solution for this, set this at the bottom of the theme.css file
```
.scrollbars {
    overflow: auto;
    height: 150px !important; /* Sets the height to show the image */
}
```
This will let you upload one photo, this will also change the settings for the activity feed as well

**Comment by Elshara** [comment]
Nice!

**Comment by aslafy-z** [comment]
Hi, @Elshara Why i was mentioned here?

**Comment by Elshara** [comment]
:) I group tagged people who were involved with git hub since 2015
relating to social engine php in some way. The discussion of this
issue admittedly became somewhat off topic and I wanted to follow it
up with a mention of what all we have contributed to for the past 3
years. It may not seem like much, but I wanted you to know your
importance and value is sincere and I've made a few friends with
people who haven't posted here for awhile. I wanted to say hi and to
let you know you are worth the appreciation you deserve, since we're
all a kind of team. That's what brought up my interest in making it
known as best I could.
I apologize if it bothered you.

**Comment by brettneese** [comment]
...I have no idea what social engine is.

**Comment by ayhanbastan** [comment]
thanks.. :)


________________________________
Gönderen: Elshara Silverheart <notifications@github.com>
Gönderildi: 1 Mayıs 2017 Pazartesi 12:32
Kime: SocialEngine/phpv4-issues
Bilgi: ayhanbastan; Mention
Konu: Re: [SocialEngine/phpv4-issues] cant post photo in inbox (#699)

:) I group tagged people who were involved with git hub since 2015
relating to social engine php in some way. The discussion of this
issue admittedly became somewhat off topic and I wanted to follow it
up with a mention of what all we have contributed to for the past 3
years. It may not seem like much, but I wanted you to know your
importance and value is sincere and I've made a few friends with
people who haven't posted here for awhile. I wanted to say hi and to
let you know you are worth the appreciation you deserve, since we're
all a kind of team. That's what brought up my interest in making it
known as best I could.
I apologize if it bothered you.

**Comment by DonnaScriptTechs** [comment]
Ok I see this is not resolved in 4.9.2. I thought it had been.

**Comment by DonnaScriptTechs** [comment]
This is fixed for our 4.9.3 version when we release it.
